src/components/MyKiva/LendingStats.vue

```diff
@@ -1,14 +1,90 @@
 <template>
-	<div>
-		<h3
-			class="tw-text-primary"
-		>
+	<div class="tw-mb-3.5">
+		<h3 class="tw-text-primary tw-mb-1">
 			Ready to grow your impact?
 		</h3>
-		<p
-			class="tw-text-base"
-		>
+		<p class="tw-text-base">
 			Next steps for you based on your lending history
 		</p>
 	</div>
+	<div class="tw-bg-white tw-rounded tw-shadow tw-p-1 md:tw-p-2 tw-w-full" style="max-width: 800px;">
+		<div class="tw-mb-4">
+			<!-- Pill above the list -->
+			<span
+				class="tw-inline-flex tw-items-center tw-gap-1.5 tw-mb-2 md:tw-mb-3 tw-rounded tw-bg-eco-green-1
+				tw-px-3 tw-py-1 tw-leading-tight"
+				title="Your lending reach"
+			>
+				<GlobeSearch class="tw-w-3 tw-h-3 tw-text-brand-550 tw-align-middle" />
+				<span class="tw-text-primary" tw-font-medium>
+					{{ loanRegions }}/{{ totalRegions }} Regions supported
+				</span>
+			</span>
+			<div class="tw-flex tw-flex-col md:tw-flex-row tw-gap-y-2 md:tw-gap-x-6">
+				<ul
+					class="tw-grid tw-grid-cols-2 sm:tw-grid-cols-3 md:tw-grid-cols-4
+						tw-gap-y-2 tw-gap-x-2 tw-w-full md:[&>li]:tw-max-w-[170x]"
```

Comment:
Why is `md:[&>li]:tw-max-w-[170x]` needed? Also we'll want to replace this JIT TW class.